org.davmail.DavMail.json

```diff
@@ -0,0 +1,83 @@
+{
+  "app-id": "org.davmail.DavMail",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "1.6",
```

Comment:
This should now be changed to 18.08 (OpenJDK on flathub has been updated to support that)

Reply:
Updated to 18.08